main.tf

```diff
@@ -1,7 +1,8 @@
 locals {
-  naming_suffix       = "internal-tableau-${var.naming_suffix}"
-  naming_suffix_linux = "internal-tableau-linux-${var.naming_suffix}"
-  naming_suffix_wip   = "tableau-linux-${var.naming_suffix}"
+  naming_suffix         = "internal-tableau-${var.naming_suffix}"
+  naming_suffix_linux   = "internal-tableau-linux-${var.naming_suffix}"
+  naming_suffix_wip     = "tableau-linux-${var.naming_suffix}"
+  naming_suffix_wip_pos = "tableau-${var.naming_suffix}"
```

Comment:
What does `pos` stand for? Why there are 2 `wip` locals?

Reply:
"pos" stands for posgres (rds). The ec2 and rds will be called "ec2-wip-tableau-linux-apps-notprod-dq" and "wip-postgres-tableau-apps-notprod-dq". So we will need a naming suffix that appends "tableau-linux-" and one that appends "tableau-". We could change the var names to simply "naming_suffix_wip" and "naming_suffix_wip_linux".

Reply:
Okay thanks for that. Can I suggest to change the RDS name to `postgres-tableau-wip-apps-notprod-dq` instead? So that it is inline with the other RDS instances.

Reply:
As discussed I'll mark this as resolved.